core/templates/dev/head/services/stateful/FocusManagerServiceSpec.js

```diff
@@ -0,0 +1,76 @@
+// Copyright 2016 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for the FocusManagerService.
+ */
+
+describe('Focus Manager Service', function() {
+  var FocusManagerService;
+  var DeviceInfoService;
+  var IdGenerationService;
+  var clearFocusLabel;
+  var rootScope;
+  var $timeout;
+  var focuslabel = 'FocusLabel';
+
+  beforeEach(module('oppia'));
+  beforeEach(inject(function($injector, LABEL_FOR_CLEARING_FOCUS) {
```

Comment:
@seanlip, should `$provide` service be used to get the constant `LABEL_FOR_CLEARING_FOCUS`?

Reply:
I would do $injector.get(), same as for all the others. Use $provide only if you want to mock it.

Reply:
Done